packages/react/src/LogoSuite/LogoSuite.stories.tsx

```diff
@@ -130,3 +136,17 @@ export const Playground = args => (
     </LogoSuite.Logobar>
   </LogoSuite>
 )
+
+export const WithPngLogos = args => (
+  <LogoSuite {...args}>
+    <LogoSuite.Heading visuallyHidden={args.visuallyHidden}>Heading</LogoSuite.Heading>
+    <LogoSuite.Description>Body text maximus ligula felis, non egestas dolor rutrum vel.</LogoSuite.Description>
+    <LogoSuite.Logobar variant={args.variant} marquee={args.hasMarquee} marqueeSpeed={args.marqueeSpeed}>
+      <Image alt="Uber" src={uberLogo} />
+      <Image alt="Vercel" src={vercelLogo} />
+      <Image alt="Shopify" src={shopifyLogo} />
+      <Image alt="Pinterest" src={pinterestLogo} />
+      <Image alt="Twilio" src={twilioLogo} />
+    </LogoSuite.Logobar>
+  </LogoSuite>
+)
```

Comment:
Sorry, only just spotted this but could you add this story to LogoSuite.features.stories.tsx instead please? All of the stories for component-specific features live in there 🙂

Reply:
<picture data-single-emoji=":facepalm:" title=":facepalm:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T05CSQFHRAP/facepalm/dc224762cb1dd67e.png" alt=":facepalm:" align="absmiddle"></picture> Good catch. Changed.